core/src/kvs/ds.rs

```diff
@@ -820,25 +860,109 @@ impl Datastore {
 	// It is handy for testing, because it allows you to specify the timestamp,
 	// without depending on a system clock.
 	pub async fn tick_at(&self, ts: u64) -> Result<(), Error> {
-		self.save_timestamp_for_versionstamp(ts).await?;
+		let _vs = self.save_timestamp_for_versionstamp(ts).await?;
```

Comment:
I assume this was for debugging? Don't see this variable being referenced anywhere

Reply:
It's now being returned out of the function. The value may be useful. It's not used, but removing it seems backward. If we are saving a timestamp for a versions stamp, we want to know what versionstamp we had. I would like to keep it as is if that's ok.

Reply:
Well it's not returned right? haha
But from what I understand is that you want to use it and not forget about it in a future PR?